app/models/ExportsDeclaration.scala

```diff
@@ -63,6 +76,9 @@ case class ExportsDeclaration(
   def clearRoutingCountries(): ExportsDeclaration =
     copy(locations = locations.copy(hasRoutingCountries = Some(false), routingCountries = Seq.empty))
 
+  def commodityCode(itemId: String): Option[String] =
+    itemBy(itemId).flatMap(_.commodityDetails.flatMap(_.combinedNomenclatureCode))
```

Comment:
👍

Reply:
This method already exists `getCommodityCodeOfItem` Line 108